lib/filesettings.h

```diff
@@ -59,6 +59,16 @@ class FileWithDetails
     {
         return mSize;
     }
+
+    std::string updateSize()
```

Comment:
It would be better to use a setter function so we are consistent with the rest of the code (well, the upcoming rest of the code).

Reply:
Ah - I now see why we are doing that. So please disregard my comment.